### PR TITLE
initial eol announcement for 1.17

### PR DESCRIPTION
**Pull request by dhawton**
Please provide a description for what this PR is for.

Initial 1.17 EOL announcement

cc: @pmerrison @johnma14 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/support/announcing-1.17/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.17 ends on October 27, 2023
+subtitle: Support Announcement
+description: Upcoming Istio 1.16 end of life announcement.
```
Comment (ericvn) [inline content/en/news/support/announcing-1.17/index.md:4]:
```suggestion
description: Upcoming Istio 1.17 end of life announcement.
```

File: content/en/news/support/announcing-1.17/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+publishdate: 2023-10-27
```
Comment (ericvn) [inline content/en/news/support/announcing-1.17/index.md:5]:
```suggestion
publishdate: 2023-9-27
```

File: content/en/news/support/announcing-1.17/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), minor releases like 1.17 are supported until six weeks after the N+2 minor release (1.19 in this case). [Istio 1.19 was released on September 5th, 2023](/news/releases/1.18.x/announcing-1.18/), and support for 1.17 will end on October 27th, 2023.
```
Comment (ericvn) [inline content/en/news/support/announcing-1.17/index.md:8]:
```suggestion
According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), minor releases like 1.17 are supported until six weeks after the N+2 minor release (1.19 in this case). [Istio 1.19 was released on September 5th, 2023](/news/releases/1.19.x/announcing-1.19/), and support for 1.17 will end on October 27th, 2023.
```

**Comment by dhawton** [comment]
I swear I changed all of those. :D

File: content/en/news/support/announcing-1.17/index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.17 ends on October 27, 2023
+subtitle: Support Announcement
+description: Upcoming Istio 1.17 end of life announcement.
+publishdate: 2023-9-27
```
Comment (ericvn) [inline content/en/news/support/announcing-1.17/index.md:5]: I'm not seeing this on the deploy details. I'm wondering if the date needs to be 2023-09-27.

I'd also change the directory to content/en/news/support/announcing-1.17-eol

Comment (dhawton) [reply content/en/news/support/announcing-1.17/index.md]: It was the 0

File: content/en/news/support/announcing-1.17-eol/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.17 ends on October 27, 2023
+subtitle: Support Announcement
+description: Upcoming Istio 1.17 end of life announcement.
+publishdate: 2023-09-27
+---
+
+According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), minor releases like 1.17 are supported until six weeks after the N+2 minor release (1.19 in this case). [Istio 1.19 was released on September 5th, 2023](/news/releases/1.19.x/announcing-1.19/), and support for 1.17 will end on October 27th, 2023.
```
Comment (ericvn) [inline content/en/news/support/announcing-1.17-eol/index.md:8]: Probably starting to get too picky...
```suggestion
According to Istio's [support policy](/docs/releases/supported-releases#support-policy/), minor releases like 1.17 are supported until six weeks after the N+2 minor release (1.19 in this case). [Istio 1.19 was released on September 5th, 2023](/news/releases/1.19.x/announcing-1.19/), and support for 1.17 will end on October 27th, 2023.
```

File: content/en/news/support/announcing-1.17-eol/index.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.17 ends on October 27, 2023
+subtitle: Support Announcement
+description: Upcoming Istio 1.17 end of life announcement.
+publishdate: 2023-09-27
+---
+
+According to Istio's [support policy](/docs/releases/supported-releases#support-policy/), minor releases like 1.17 are supported until six weeks after the N+2 minor release (1.19 in this case). [Istio 1.19 was released on September 5th, 2023](/news/releases/1.19.x/announcing-1.19/), and support for 1.17 will end on October 27th, 2023.
```
[the remark this answers is not in the document]
Comment (dhawton) [reply content/en/news/support/announcing-1.17-eol/index.md:8]: Odd, that slash has been there for a few years now... but good to remove it anyway.

**Review by ericvn — APPROVED** [review]

**Pull request merged**